lib/webpacker/commands.rb

```diff
@@ -11,6 +11,7 @@ def clobber
   end
 
   def bootstrap
+    return unless config.config_path.exist?
```

Comment:
I don't think we should have any check inside the method of #bootstrap. I'd rather have it where #bootstrap is called. Also don't want to do any puts anywhere inside these classes. They should all use the logger. Only the rake tasks can put.